src/api/serviceCatalog.js

```diff
@@ -0,0 +1,24 @@
+/*
+To fetch the service catalog from the API:
+*/
+import { SERVICE_CATALOG_API } from '../config/apiPaths';
+import { BOOKING_TYPE_DOCTOR } from '../config/constants';
+import { easyGet } from './easyFetch';
+const FILENAME = 'api/serviceCatalog';
+
+export async function fetchCatalog(bookingType) {
+  const url = bookingType === BOOKING_TYPE_DOCTOR ? '/doctors' : '/lab-tests';
+  const response = await easyGet({
+    url: SERVICE_CATALOG_API + url,
+  });
+  if (response.done) {
+    console.log(`${FILENAME} - Fetching service catalog`);
+    return response.body.data.items;
+  } else if (response.userError) {
+    console.error(`${FILENAME} - Error fetching service catalog: ${response.body}`);
+    return null;

```

Comment:
change `response.body` to `null` or something
Component should know that call failed